### ISSUE TITLE
Add support for loading indicators

### ISSUE THREAD

**Issue by imacrayon** [imacrayon/alpine-ajax#6]
There's a need to add loading indicators when network requests are slow. I think this will require a new directive. My initial thoughts are that it would work like this:
```html
<template x-busy>
<img src="loading.svg" alt="Loading">
</template>
```
This template content would show only when AJAX is processing a network request. I'm open to bikeshedding other attribute names other than `x-busy`.

**Comment by imacrayon** [comment]
I've landed on adding the `aria-busy` attribute to elements while a request is in progress. Combining the presence of `aria-busy` with some CSS should allow developers to build simple loading indicators. Here's some CSS I've been playing with:
```
[aria-busy] {
  --stroke-width: 6px;
  --size: 64px;
  --duration: 1s;
  opacity: .75
}
[aria-busy]:before {
  content: '';
  position: absolute;
  top: 50%;
  left: 50%;
  width: var(--size);
  height: var(--size);
  margin-top: calc(var(--size) / 2 * -1);
  margin-left: calc(var(--size) / 2 * -1);
  border: var(--stroke-width) solid rgba(0, 0, 0, 0.15);
  border-radius: 50%;
  border-top-color: rgba(0, 0, 0, 0.5);
  animation: rotate calc(var(--duration)) linear infinite;
}
@keyframes rotate {
  0% { transform: rotate(0deg); }
  100% { transform: rotate(360deg); }
}

```

This will be released in 0.0.20

**Comment by imacrayon** [comment]
Added in 9d7bfe074ffd2ee3512489792bcb9bdf1d4e06af
